### ISSUE TITLE
installation error

### ISSUE THREAD

**Issue by zeke** [rhysd/Chromenu#2]
```
Last login: Sat Jan  7 16:33:13 on ttys000
You have new mail.

~ master*
❯ npm install -g chromenu
npm http request GET https://registry.npmjs.org/chromenu
npm http 200 https://registry.npmjs.org/chromenu
npm http fetch GET https://registry.npmjs.org/chromenu/-/chromenu-1.1.2.tgz
npm http fetch 200 https://registry.npmjs.org/chromenu/-/chromenu-1.1.2.tgz
npm http request GET https://registry.npmjs.org/bulma
npm http request GET https://registry.npmjs.org/electron-in-page-search
npm http request GET https://registry.npmjs.org/heredocument
npm http request GET https://registry.npmjs.org/react-component-bytesize-icons
npm http request GET https://registry.npmjs.org/electron
npm http request GET https://registry.npmjs.org/font-awesome
npm http request GET https://registry.npmjs.org/loglevel
npm http request GET https://registry.npmjs.org/cheerio
npm http request GET https://registry.npmjs.org/immutable
npm http request GET https://registry.npmjs.org/mousetrap
npm http request GET https://registry.npmjs.org/page-icon
npm http request GET https://registry.npmjs.org/react
npm http request GET https://registry.npmjs.org/react-addons-shallow-compare
npm http request GET https://registry.npmjs.org/react-redux
npm http request GET https://registry.npmjs.org/react-dom
npm http request GET https://registry.npmjs.org/redux
npm http request GET https://registry.npmjs.org/request
npm http 200 https://registry.npmjs.org/bulma
npm http fetch GET https://registry.npmjs.org/bulma/-/bulma-0.2.3.tgz
npm http fetch 200 https://registry.npmjs.org/bulma/-/bulma-0.2.3.tgz
npm http 200 https://registry.npmjs.org/mousetrap
npm http 200 https://registry.npmjs.org/loglevel
npm http 200 https://registry.npmjs.org/page-icon
npm http 200 https://registry.npmjs.org/font-awesome
npm http 200 https://registry.npmjs.org/react-component-bytesize-icons
npm http 200 https://registry.npmjs.org/electron-in-page-search
npm http 200 https://registry.npmjs.org/electron
npm http fetch GET https://registry.npmjs.org/loglevel/-/loglevel-1.4.1.tgz
npm http fetch GET https://registry.npmjs.org/font-awesome/-/font-awesome-4.7.0.tgz
npm http 200 https://registry.npmjs.org/immutable
npm http 200 https://registry.npmjs.org/cheerio
npm http 200 https://registry.npmjs.org/redux
npm http 200 https://registry.npmjs.org/heredocument
npm http fetch GET https://registry.npmjs.org/electron-in-page-search/-/electron-in-page-search-1.2.3.tgz
npm http fetch GET https://registry.npmjs.org/react-component-bytesize-icons/-/react-component-bytesize-icons-2.0.1.tgz
npm http 200 https://registry.npmjs.org/request
npm http 200 https://registry.npmjs.org/react-addons-shallow-compare
npm http 200 https://registry.npmjs.org/react-dom
npm http fetch 200 https://registry.npmjs.org/loglevel/-/loglevel-1.4.1.tgz
npm http 200 https://registry.npmjs.org/react-redux
npm http fetch 200 https://registry.npmjs.org/electron-in-page-search/-/electron-in-page-search-1.2.3.tgz
npm http 200 https://registry.npmjs.org/react
npm http fetch 200 https://registry.npmjs.org/react-component-bytesize-icons/-/react-component-bytesize-icons-2.0.1.tgz
npm http fetch GET https://registry.npmjs.org/heredocument/-/heredocument-2.0.2.tgz
npm http fetch GET https://registry.npmjs.org/react-addons-shallow-compare/-/react-addons-shallow-compare-15.4.2.tgz
npm http fetch GET https://registry.npmjs.org/react-dom/-/react-dom-15.4.2.tgz
npm http fetch GET https://registry.npmjs.org/react-redux/-/react-redux-5.0.1.tgz
npm http fetch GET https://registry.npmjs.org/react/-/react-15.4.2.tgz
npm http fetch 200 https://registry.npmjs.org/heredocument/-/heredocument-2.0.2.tgz
npm http fetch 200 https://registry.npmjs.org/react-addons-shallow-compare/-/react-addons-shallow-compare-15.4.2.tgz
npm http fetch 200 https://registry.npmjs.org/font-awesome/-/font-awesome-4.7.0.tgz
npm http fetch 200 https://registry.npmjs.org/react-dom/-/react-dom-15.4.2.tgz
npm http fetch 200 https://registry.npmjs.org/react-redux/-/react-redux-5.0.1.tgz
npm http fetch 200 https://registry.npmjs.org/react/-/react-15.4.2.tgz
npm ERR! git rev-list -n1 ensure-to-launch-app: fatal: ambiguous argument 'ensure-to-launch-app': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 ensure-to-launch-app: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 ensure-to-launch-app: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 ensure-to-launch-app: 
npm ERR! git rev-list -n1 ensure-to-launch-app: fatal: ambiguous argument 'ensure-to-launch-app': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 ensure-to-launch-app: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 ensure-to-launch-app: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 ensure-to-launch-app: 
npm ERR! git clone --template=/Users/zeke/.npm/_git-remotes/_templates --mirror git@github.com:rhysd/menubar.git /Users/zeke/.npm/_git-remotes/git-github-com-rhysd-menubar-git-ensure-to-launch-app-bd46afc9: Cloning into bare repository '/Users/zeke/.npm/_git-remotes/git-github-com-rhysd-menubar-git-ensure-to-launch-app-bd46afc9'...
npm ERR! git clone --template=/Users/zeke/.npm/_git-remotes/_templates --mirror git@github.com:rhysd/menubar.git /Users/zeke/.npm/_git-remotes/git-github-com-rhysd-menubar-git-ensure-to-launch-app-bd46afc9: Permission denied (publickey).
npm ERR! git clone --template=/Users/zeke/.npm/_git-remotes/_templates --mirror git@github.com:rhysd/menubar.git /Users/zeke/.npm/_git-remotes/git-github-com-rhysd-menubar-git-ensure-to-launch-app-bd46afc9: fatal: Could not read from remote repository.
npm ERR! git clone --template=/Users/zeke/.npm/_git-remotes/_templates --mirror git@github.com:rhysd/menubar.git /Users/zeke/.npm/_git-remotes/git-github-com-rhysd-menubar-git-ensure-to-launch-app-bd46afc9: 
npm ERR! git clone --template=/Users/zeke/.npm/_git-remotes/_templates --mirror git@github.com:rhysd/menubar.git /Users/zeke/.npm/_git-remotes/git-github-com-rhysd-menubar-git-ensure-to-launch-app-bd46afc9: Please make sure you have the correct access rights
npm ERR! git clone --template=/Users/zeke/.npm/_git-remotes/_templates --mirror git@github.com:rhysd/menubar.git /Users/zeke/.npm/_git-remotes/git-github-com-rhysd-menubar-git-ensure-to-launch-app-bd46afc9: and the repository exists.
npm ERR! Darwin 16.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "chromenu"
npm ERR! node v7.2.1
npm ERR! npm  v3.10.10
npm ERR! code 128

npm ERR! Command failed: git clone --template=/Users/zeke/.npm/_git-remotes/_templates --mirror git@github.com:rhysd/menubar.git /Users/zeke/.npm/_git-remotes/git-github-com-rhysd-menubar-git-ensure-to-launch-app-bd46afc9
npm ERR! Cloning into bare repository '/Users/zeke/.npm/_git-remotes/git-github-com-rhysd-menubar-git-ensure-to-launch-app-bd46afc9'...
npm ERR! Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/zeke/npm-debug.log
```

cc @rhysd

**Comment by rhysd** [comment]
I found one dependant npm package spec was changed. I'll fix. The error did not occur in my environment because of npm pacakages cache. But I could find the reason from your log :+1:

**Comment by rhysd** [comment]
I fixed problem and released v1.1.3 as npm pacakge. It also includes dependency updates. If you find the problem is still there, please let me know by commenting here.

**Comment by zeke** [comment]
Fixed. Thanks!
